### ISSUE TITLE
Support SwiftPM Package.pins

### ISSUE THREAD

**Issue by andrew** [librariesio/libraries.io#974]
https://github.com/apple/swift-package-manager/blob/master/Documentation/Usage.md#package-pinning

JSON file called `Package.pins` which is generated/updated automatically as a lockfile 

Only example I can find so far is here: https://github.com/apple/swift-package-manager/blob/master/Tests/WorkspaceTests/PinsStoreTests.swift#L99-L122

```json
{
  "autoPin": true,
  "pins": [
    {
      "package": "bam",
      "reason": null,
      "repositoryURL": "/private/tmp/BigPackage/bam",
      "version": "1.0.0"
    },
    {
      "package": "bar",
      "reason": null,
      "repositoryURL": "/private/tmp/BigPackage/bar",
      "version": "1.0.0"
    },
    {
      "package": "baz",
      "reason": null,
      "repositoryURL": "/private/tmp/BigPackage/baz",
      "version": "1.0.0"
    }
  ],
  "version": 1
}
```

**Comment by andrew** [comment]
It's shipped in swift 3.1 so no longer on hold: https://github.com/kumabook/TOMLKit/commit/6a7df51a4962771bd75c19c91948f2f3238f19ed

**Comment by andrew** [comment]
Moving this to the [Backlog](https://github.com/librariesio/libraries.io/blob/master/Backlog.md) as we'd still like to implement it but can't see that happening in the near future.
